src/providers/data_provider_be.c

```diff
@@ -126,11 +145,12 @@ void be_mark_offline(struct be_ctx *ctx)
         DEBUG(SSSDBG_TRACE_INTERNAL, "Initialize check_if_online_ptask.\n");
 
         offline_timeout = get_offline_timeout(ctx);
+        offline_timeout = get_offline_timeout_max(ctx);
```

Comment:
I think here should be offline_timeout_max rather than offline_timeout :)

Reply:
Oops! Fixed :-)